packages/cli/src/commands/doctor/healthchecks/common.js

```diff
@@ -22,13 +22,13 @@ const logManualInstallation = ({
 
   if (url) {
     return logMessage(
-      `Read more about how to download ${healthcheck} at ${chalk.dim(url)}`,
+      `Read more about how to download ${healthcheck} at ${chalk.bold(url)}`,
```

Comment:
We `chalk.dim.underline` the links, can we keep it consistent?

Reply:
Good eye, fixed.